app/assets/javascripts/components/settings/views/salesforce_credentials_form.jsx

```diff
@@ -1,57 +1,54 @@
-import createReactClass from 'create-react-class';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 import TextInput from '../../common/text_input';
 
-const SalesforceCredentialsForm = createReactClass({
-  propTypes: {
-    updateSalesforceCredentials: PropTypes.func,
-    handlePopoverClose: PropTypes.func,
-  },
-
-  getInitialState() {
-    return {};
-  },
-
-  handleChange(key, value) {
-    return this.setState({ [key]: value });
-  },
-
-  handleSubmit(e) {
+const SalesforceCredentialsForm = (props) => {
```

Comment:
```suggestion
const SalesforceCredentialsForm = ({updateSalesforceCredentials, handlePopoverClose}) => {
```
I think destructuring the props object is a nice change to have when doing these components refactors. Besides that, everything looks good!

Reply:
Done on d0d6562260628ad79f1bf69d437b49de10b59b95